content/en/blog/2019/trustworthy-jwt-sds/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Istio 1.3 with SDS Announcement
+description: Istio 1.3 with SDS Announcement.
+publishdate: 2019-08-23
+attribution: Phillip Quy Le (Google)
+keywords: [security, PKI, certificate, nodeagent, sds]
+---
+
+In Istio 1.3, we have made a change to take advantage of improvements in Kubernetes in order to more
+securely issue certificates for workloads.
+
+When a Citadel Agent (the agent that runs on a node and is responsible for obtaining certificates)
+sends a Certificate Signing Request to Citadel to get a certificate for a workload,
```

Comment:
I couldn't find `Certificate Signing Request` in that medium post. I would suggest we don't even capitalize SDS. The convention is not to capitalize the words of the abbreviation. See https://en.wikipedia.org/wiki/Certificate_signing_request
Be wary of using capitalization for emphasis, that should be avoided because readers will be confused by it.

Reply:
Got it. Thanks!